src/components/board-grid.tsx

```diff
@@ -13,14 +17,37 @@ const game: { gameboard: null[][] } = {
 };
 
 export function BoardGrid() {
+  const { gameId } = useParams();
+  const [theGame, isLoading] = useRunningGame(gameId!);
+
+  if (!theGame) {
+    return <p>"Loading"</p>;
+  }
+  const { players } = theGame;
+  const player1 = players[0];
+  const player1name = Object.values(player1)[0];
```

Comment:
This will give you an array of all values in the object, which just **happens** to be of length 1 where the name is first.

However should you want to add more to the player object in the future this will break

I would rather do 
```
const [{name: player1Name}, {...}] = players
```